test/unit/mem/view/src/ViewStaticAccMem.cpp

```diff
@@ -16,92 +16,119 @@ using Elem = std::uint32_t;
+                    + offset));
+    }
+};
 
+#if defined(ALPAKA_ACC_SYCL_ENABLED) && defined(ALPAKA_SYCL_TARGET_CPU)
+template<typename TDim, typename TIdx>
+using EnabledAccsElseInt = std::tuple<alpaka::AccCpuSycl<TDim, TIdx>>;
+template<typename TDim, typename TIdx>
+using EnabledAccs = typename alpaka::meta::Filter<EnabledAccsElseInt<TDim, TIdx>, std::is_class>;
+using TestAccs = EnabledAccs<Dim, Idx>;
+#elif defined(ALPAKA_ACC_SYCL_ENABLED) && defined(ALPAKA_SYCL_TARGET_GPU)
+template<typename TDim, typename TIdx>
+using EnabledAccsElseInt = std::tuple<alpaka::AccGpuSyclIntel<TDim, TIdx>>;
+template<typename TDim, typename TIdx>
+using EnabledAccs = typename alpaka::meta::Filter<EnabledAccsElseInt<TDim, TIdx>, std::is_class>;
+using TestAccs = EnabledAccs<Dim, Idx>;
+#else
 using TestAccs = alpaka::test::EnabledAccs<Dim, Idx>;
+#endif
 
 TEMPLATE_LIST_TEST_CASE("staticDeviceMemoryGlobal", "[viewStaticAccMem]", TestAccs)
 {
-#if !defined(ALPAKA_ACC_SYCL_ENABLED)
     using Acc = TestType;
     using DevAcc = alpaka::Dev<Acc>;
 
     auto const platformAcc = alpaka::Platform<Acc>{};
+
     auto const devAcc = alpaka::getDevByIdx(platformAcc, 0);
 
     alpaka::Vec<Dim, Idx> const extent(3u, 2u);
 
-    alpaka::test::KernelExecutionFixture<Acc> fixture(extent);
-
-    StaticDeviceMemoryTestKernel kernel;
-
-    // uninitialized static constant device memory
+    // uninitialized static global device memory
     {
         auto const platformHost = alpaka::PlatformCpu{};
         auto const devHost = alpaka::getDevByIdx(platformHost, 0);
 
         using QueueAcc = alpaka::test::DefaultQueue<DevAcc>;
         QueueAcc queueAcc(devAcc);
 
+        alpaka::test::KernelExecutionFixture<Acc> fixture(queueAcc, extent);
+
+        StaticDeviceMemoryTestKernel kernel;
+
         std::vector<Elem> const data{0u, 1u, 2u, 3u, 4u, 5u};
         auto bufHost = alpaka::createView(devHost, data.data(), extent);
 
-        auto viewConstantMemUninitialized
-            = alpaka::createStaticDevMemView(&g_constantMemory2DUninitialized[0u][0u], devAcc, extent);
-
-        alpaka::memcpy(queueAcc, viewConstantMemUninitialized, bufHost);
+        alpaka::memcpy(
+            queueAcc,
+            g_globalMemory2DUninitialized<typename alpaka::trait::AccToTag<Acc>::type>,
+            bufHost,
+            extent);
         alpaka::wait(queueAcc);
 
-        REQUIRE(fixture(kernel, alpaka::getPtrNative(viewConstantMemUninitialized)));
-    }
+        REQUIRE(fixture(kernel));
 
-#else // !defined(ALPAKA_ACC_SYCL_ENABLED)
-
-    WARN("The SYCL backend does not support global device variables.");
-
-#endif // !defined(ALPAKA_ACC_SYCL_ENABLED)
+        std::vector<Elem> data2(6, 0u);
+        auto bufHost2 = alpaka::createView(devHost, data2.data(), extent);
+        alpaka::memcpy(
+            queueAcc,
+            bufHost2,
+            g_globalMemory2DUninitialized<typename alpaka::trait::AccToTag<Acc>::type>,
```

Comment:
I would prefer passing the `Acc` type directly instead using the type trait `alpaka::trait::AccToTag`. I think the common case is, that we have the `Acc` type available and not the tag type, therefore we would have a lot of copy/past code.

Depending on if it makes sense to provide the possibility to use tags in the user API, you can ether move `typename alpaka::trait::AccToTag<Acc>::type` to the implementation (no support for tags in the user API) or do some kind of overload and check if the given type is a alpaka accelerator type or tag.

Reply:
I've moved it to the implementation
```cpp
template<typename TAcc, typename T>
    using DevGlobal = typename detail::DevGlobalTrait<typename alpaka::trait::AccToTag<TAcc>::type, T>::Type;
```